src/main/java/Event.java

```diff
@@ -0,0 +1,20 @@
+public class Event extends Task {
+    protected String timings;
```

Comment:
not sure in your context, but if only 1 timing then consider using singular. Plurals are for arrays and lists.